lib/tasks/add_countries.rake

```diff
@@ -0,0 +1,37 @@
+namespace :add_countries do
+  desc "Add countries to support interface"
+  task generate: :environment do
+    countries_hash.each do |countries|
+      code = countries[:code]
+      regions = countries[:regions]
+      country = Country.find_or_create_by!(code:)
+
+      if regions.blank?
+        country
+          .regions
+          .find_or_create_by!(name: "")
+          .update!(legacy: false, application_form_enabled: false)
+      else
+        regions.each do |region|
+          next if country.regions.where(name: region[:name]).any?
+
+          country.regions.create!(region)
```

Comment:
```suggestion
          country.regions.create!(legacy: false, application_form_enabled: false, **region)
```

Perhaps we want to default to this as well like the "national" region in the lines above?